cmd/skopeo/utils.go

```diff
@@ -44,12 +47,33 @@ func sharedImageFlags() ([]cli.Flag, *sharedImageOptions) {
 	return []cli.Flag{
 		cli.StringFlag{
 			Name:        "authfile",
-			Usage:       "path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json",
+			Value:       defaultAuthFile(),
+			Usage:       "path of the authentication file.",
```

Comment:
Why isn’t this `authFileUsage`?

Reply:
I guess it could be.  It will do the exact same thing and look the same.

Reply:
No it would not. The issue is setting the Value field causes the CLI help information to expand to  path of the authentication file. (default: "/run/user/3267/containers/auth.json") (default: "/run/user/3267/containers/auth.json")
IE It would put the message out twice.

Reply:
I have changed the code to show the expected default value, but iff XDG_RUNTIME_DIR is not set and the user does not specify a value, then containers/image will still get "" so it can figure out its own default value.